Library/Formula/fits.rb

```diff
@@ -1,11 +1,9 @@
-require 'formula'
-
 class Fits < Formula
   desc "File Information Tool Set"
-  homepage 'http://fitstool.org/'
-  url 'http://projects.iq.harvard.edu/files/fits/files/fits-0.8.4_0.zip'
-  version '0.8.4'
-  sha1 'f9b6b13cf1e818c6cdcfec71eb3dcd6804dd8819'
+  homepage "http://projects.iq.harvard.edu/fits"
+  url "http://projects.iq.harvard.edu/files/fits/files/fits-0.8.6_1.zip"
```

Comment:
This `url` does support https (without redirection):
https://projects.iq.harvard.edu/files/fits/files/fits-0.8.6_1.zip

The `homepage` as well, but it redirects to plaintext.